bokehjs/test/models/plots/plot.coffee

```diff
@@ -190,6 +190,35 @@ describe "Plot.View render", ->
     @test_plot_view.render()
     expect(spy.calledOnce).to.be.true
 
+  it "canvas.ctx attribute should exist to allow tools to grab the DOM canvas", ->
+    expect(@test_plot_view.canvas_view.ctx).to.exist
+
+
+describe "Plot.View get_canvas_element", ->
+
+  afterEach ->
+    utils.unstub_canvas()
+    utils.unstub_solver()
+
+  beforeEach ->
+    utils.stub_canvas()
+    utils.stub_solver()
+
+    @test_doc = new Document()
+    @test_plot = new Plot({
+      x_range: new Range1d({start: 0, end: 1})
+      y_range: new Range1d({start: 0, end: 1})
+    })
+    @test_plot.document = @test_doc
+    @test_plot._doc_attached()
+    @test_plot_view = new @test_plot.default_view({ 'model': @test_plot })
+
+  it "should exist because get_canvas_element depends on it", ->
+    expect(@test_plot_view.canvas_view.ctx).to.exist
```

Comment:
Does exist actually test for anything - I can't make it tell me anything useful. If I delete something that I was expecting to exist then it still passes. I'm not sure, I just want you to be sure that you've seen these tests failing and passing in a predictable way.